pom.xml

```diff
@@ -164,8 +164,7 @@
         <spring.ldap.core>2.4.1</spring.ldap.core>
         <janus.version>0.6.2</janus.version>
         <spring-boot.version>2.7.5</spring-boot.version>
-        <janus.version>0.6.1</janus.version>
-        <gremlin.version>3.5.2</gremlin.version>
+        <gremlin.version>3.6.1</gremlin.version>
```

Comment:
shouldn't this name be tinkerpop.version in line with the way janus references this value?

Reply:
yes agree, I will change the variable to tinkerpop.version